plugins/inputs/systemd_units/subcommand_list.go

```diff
@@ -0,0 +1,88 @@
+//go:generate ../../../tools/readme_config_includer/generator
+package systemd_units
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"strings"
+
+	"github.com/influxdata/telegraf"
+)
+
+// Gather parses systemctl outputs and adds counters to the Accumulator
+func parseListUnits(_ *SystemdUnits, buffer *bytes.Buffer, acc telegraf.Accumulator) error {
+	scanner := bufio.NewScanner(buffer)
+	for scanner.Scan() {
+		line := scanner.Text()
+
+		data := strings.Fields(line)
+		if len(data) < 4 {
+			acc.AddError(fmt.Errorf("Error parsing line (expected at least 4 fields): %s", line))
```

Comment:
Error messages should start lowercase so how about
```suggestion
			acc.AddError(fmt.Errorf("parsing line failed (expected at least 4 fields): %s", line))
```
Same for all other error messages here.